tests/scripts/all.sh

```diff
@@ -186,6 +187,7 @@ Special options:
   --list-components     List components supported on this platform and exit.
 
 General options:
+  -q|--quiet            Only output component names, and errors if any.
```

Comment:
Why did you add `--quiet` in this PR, since you aren't using it for the purpose of this PR?

Reply:
This is just work-in-progress that I pushed mainly so that the code doesn't exist only on my laptop. It's not complete yet, and obviously the intention behind `--quiet` is to use it in the hook.